_inc/jetpack-server-sandbox.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * This feature is only useful for Automattic developers.
+ * It configures Jetpack to talk to staging/sandbox servers
+ * on WordPress.com instead of production servers.
+ */
+
+/**
+ * @param string $sandbox Sandbox domain
+ * @param string $url URL of request about to be made
+ * @param array  $headers Headers of request about to be made
+ * @return array [ 'url' => new URL, 'host' => new Host ]
+ */
+function jetpack_server_sandbox_request_parameters( $sandbox, $url, $headers ) {
+	$host = '';
+
+	$url_host = parse_url( $url, PHP_URL_HOST );
+
+	switch ( $url_host ) {
+	case 'public-api.wordpress.com' :
+	case 'jetpack.wordpress.com' :
+	case 'dashboard.wordpress.com' :
```

Comment:
For some reason I've found the stats api was especially annoying to sandbox. Thank you for this!